Code/Framework/AzCore/AzCore/Memory/MallocSchema.cpp

```diff
@@ -11,147 +11,12 @@
 #include <AzCore/Memory/OSAllocator.h>
 #include <AzCore/std/algorithm.h>
 
-namespace AZ::Internal
-{
-    struct Header
-    {
-        uint32_t offset;
-        uint32_t size;
-    };
-} // namespace AZ::Internal
-
+ // The HphaSchemaErrorDetection.cpp file test the MallocSchema
+ // implementation with the DEBUG_ALLOCATOR define set to true
+ // In order a second implementation of this class to exist
+ // without polluting the test with using namespace ::AZStd/::AZ
+ // The implementation is placed into an inl file with no includes
```

Comment:
A copy of the previous text with same errors?

Reply:
![image](https://user-images.githubusercontent.com/56135373/180315636-a4837ada-2aeb-4527-a8d8-e23751a40ea0.png)